docs/en/userguide/index.md

```diff
@@ -0,0 +1,52 @@
+title: Session Manager
+summary: Allow members to manage and revoke access to multiple login sessions across devices.
+
+# What does the session manager module do?
+
+The session manager module allows members to manage and revoke access to multiple login sessions across devices. Each login to a member is tracked and can be viewed from the profile page. Logins can also be revoked from here.
+
+# How to use it
+
+## Getting started
+
+Make sure you have the [SilverStripe Session Manager](https://addons.silverstripe.org/add-ons/silverstripe/session-manager) module installed
+
+## Logging in
+
+When you log in with the module installed, a login session record will be created at the time of the login and associated with your current session.
```

Comment:
When logging in with "Keep me signed in for 30 days" selected, a session will remain active on that device for the full 30 days unless it is terminated prior to that allocated timeframe. Without this selected a session will only last for **_xyz_** hours. 

[notice]Note: It is advised that the "Keep me signed in" functionality is not used unless they are the only person in control of or using the device used to access the CMS.[/notice]